var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -262,7 +262,13 @@ class Hip(CMakePackage):
     patch("0014-Remove-compiler-rt-linkage-for-host-for-5.7.0.patch", when="@5.7.0:5.7")
     patch("0014-remove-compiler-rt-linkage-for-host.6.0.patch", when="@6.0:")
     patch("0015-reverting-operator-mixup-fix-for-slate.patch", when="@5.6:6.0")
-    patch("0017-Set-PARAMETERS_MIN_ALIGNMENT-to-the-native-alignment.patch", when="@5.7:6.0")
+
+    # For avx build, the start address of values_ buffer in KernelParameters is not
+    # correct as it is computed based on 16-byte alignment.
+    # https://github.com/ROCm/clr/commit/c4f773db0b4ccbbeed4e3d6c0f6bff299c2aa3f0
+    # https://github.com/ROCm/clr/commit/7868876db742fb4d44483892856a66d2993add03
+    patch("0017-Set-PARAMETERS_MIN_ALIGNMENT-to-the-native-alignment.patch", when="@5.7")
+    patch("0017-Set-PARAMETERS_MIN_ALIGNMENT-to-the-native-alignment-6.0.patch", when="@6.0")
```

Comment:
Q: Would it be possible to retrieve these patches from the URL above? Have the patches been back-ported?

Reply:
Hi @alalazo, These changes are in clr which is added as a resource.
Please let me know if there is way to patch these commit URL to resource.
https://github.com/ROCm/clr/commit/c4f773db0b4ccbbeed4e3d6c0f6bff299c2aa3f0
https://github.com/ROCm/clr/commit/7868876db742fb4d44483892856a66d2993add03

Reply:
@alalazo, Added URLs instead of using patch files. Please review. Thanks